src/components/data/data-utils.js

```diff
@@ -212,6 +213,9 @@ export const EntityUploader = ({ onSuccess, onDismiss, namespace, name, entityTy
       const workspace = Ajax().Workspaces.workspace(namespace, name)
       await (useFireCloudDataModel ? workspace.importEntitiesFile : workspace.importFlexibleEntitiesFile)(file)
       onSuccess()
+      Ajax().Metrics.captureEvent(Events.workspaceDataUpload, {
```

Comment:
Intellij gives me a warning here: "Missing await for an async function call "

Reply:
Yeah, that's on most of the mixpanel implementations, it's a known condition but isn't an issue. It happens when the mixpanel call is inside an async function even though the mixpanel event itself doesn't need to be awaited.